tools/c7n_mailer/README.md

```diff
@@ -111,6 +111,38 @@ There is a special `to` format that specifies datadog delivery, and includes the
 - metric_name: is the name of the metrics send to DataDog
 - metric_value_tag: by default the metric value send to DataDog is `1` but if you want to use one of the tags returned in the policy you can set it with the attribute `metric_value_tag`, for example in the `test-policy.yml` the value used is the size of the EBS volume. The value must be a number and it's transformed to a float value.
 
+### Slack:
+
+The Custodian mailer supports Slack messaging as a separate notification mechanism for the SQS transport method. To enable Slack integration, you must specify a Slack token in the `slack_token` field under the `mailer.yml` file.
+
+```yaml
+queue_url: https://sqs.us-east-1.amazonaws.com/1234567890/c7n-mailer-test
+role: arn:aws:iam::123456790:role/c7n-mailer-test
+slack_token: xoxo-token123
+```
+
+To enable Slack messaging, several unique fields are evaluated in the policy, as shown in the below example:
+
+```
+policies:
+  - name: c7n-mailer-test
+    resource: ebs
+    filters:
+     - Attachments: []
+    actions:
+      - type: notify
+        slack_template: slack
+        to:
+          - slack
```

Comment:
is slack too generic ? what if we want room delivery or always cc a given user? how about slack://owner ? to distinguish the current behavior?

Reply:
I'm good with this change since it sets up for future enhancements. Eventually I'd like to extend functionality to send to a particular channel as well as add support for CC'ing a particular user, etc.

Reply:
I've modified the indicated Slack target so it's more explicit, as well as including support for CC'ing a user and messaging a channel. I've updated the docs to match.